js_machine_front/src/scenes/digest/services/digest.api.ts

```diff
@@ -0,0 +1,8 @@
+import { DigestCycle } from '../../news/models/news';
+
+export async function getDigest(id: string): Promise<DigestCycle> {
+  return fetch('/getDigests', {
+    method: 'GET',
+    headers: {'Content-Type': 'application/json'},
+  }).then(response => response.json()).then(result => result.find((d: DigestCycle) => d.id === id));
```

Comment:
you can implement firebase function getDigestById or use state management to load all digests once